demo/ts/components/victory-zoom-container-demo.tsx

```diff
@@ -87,12 +82,6 @@ class CustomChart extends React.Component<any, CustomChartState> {
     };
   }
 
-  getZoomFactor() {
```

Comment:
I grepped for this and ran this in the js demo w/ a breakpoint set here and it looks like this function is never invoked

Reply:
thanks for catching this unused function, going to remove it